includes/Modules/Tag_Manager.php

```diff
@@ -452,7 +452,7 @@ protected function create_container( $account_id ) {
 
 		$client->setDefer( false );
 
-		$container = new \Google_Service_TagManager_Container();
+		$container = new \Google_Service_Tag_Manager_Container();
```

Comment:
This class is a third-party dependency and thus cannot be renamed. Can you change all occurrences of that back to its original?

Reply:
Done! A bit surprised it didn't error when the tests were run 😢